cilantro_audit/view_audit_templates.py

```diff
@@ -0,0 +1,54 @@
+import kivy
+from kivy.app import App
+from kivy.clock import Clock
+from kivy.uix.button import Button
+from kivy.uix.screenmanager import Screen
+from kivy.uix.screenmanager import ScreenManager
+from mongoengine import connect
+from cilantro_audit.constants import PROD_DB
+from cilantro_audit.constants import KIVY_REQUIRED_VERSION
+from cilantro_audit.audit_template import AuditTemplate
+
+# Required Version
+kivy.require(KIVY_REQUIRED_VERSION)
+
+# Default Database Connection
+connect(PROD_DB)
+
+
+class Manager(ScreenManager):
+    pass
+
+
+class RootPage(Screen):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        Clock.schedule_once(self.callback)
+
+    def callback(self, *args):
+        titles = list(map(lambda template: template.title, AuditTemplate.objects().only('title')))
+        for title in titles:
+            btn = AuditButton(text=title)
+            self.ids["audits_list"].add_widget(btn)
```

Comment:
Why are you not just calling the function directly here? This is the only place that `callback` is used. 

This seems like a lot of added complexity for no gain.

Also, `callback` doesn't appear to use any args, so why does it have this parameter?

And why is this function named `callback`, instead of something like `load_titles`?

Reply:
You're right, it is redundant. I just used it because I had some issues earlier but now I removed it.
Thanks

@BaderAlshaya